### DISCUSSION TITLE
Enforce Language Version 4.

### DISCUSSION THREAD

**Discussion by willflo** [netbox-community/netbox#16336] [Q&A] [answered]
Hey,

is there the possiblity to enforce the netbox ui language to english?
Since 4.0 there is the django auto translation implemented but we dont want this.
Can i disable this or enforce english for everyone?

https://netboxlabs.com/docs/netbox/en/stable/development/internationalization/

Thanks
Florian

**Comment by markkuleinio** [comment, marked as answer]
https://netboxlabs.com/docs/netbox/en/stable/configuration/system/#translation_enabled

**Reply by willflo** [reply]
Hey,

thanks
i simply just overread this point.

Thread can be closed.
